cinema/tests/test_custom_movie_viewset.py

```diff
@@ -0,0 +1,106 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient, APITestCase
+from django.contrib.auth import get_user_model
+
+from cinema.models import (
+    Movie, Genre, Actor
+)
+
+
+MOVIE_URL = reverse("cinema:movie-list")

```

Comment:
The `MOVIE_URL` variable is defined using the `reverse` function with the name `cinema:movie-list`. Ensure that this URL name is correctly defined in your URL configuration to avoid any potential issues during test execution.